lib/Frontend/SerializedDiagnosticConsumer.cpp

```diff
@@ -334,7 +302,7 @@ void SerializedDiagnosticConsumer::emitMetaBlock() {
   Stream.EnterSubblock(BLOCK_META, 3);
   Record.clear();
   Record.push_back(RECORD_VERSION);
-  Record.push_back(Version);
+  Record.push_back(VersionNumber);
```

Comment:
Note: This is 2 now, looking at git history, swift has incorrectly reported 1 since 2014. AFAICT, this is safe to change because no clients actually check it. If that's not the case, I can revert this change.

Reply:
Good catch. Let's qualify this `clang::serialized_diags::VersionNumber`

Reply:
sounds good to me, done